### PR TITLE
Next

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -2,5 +2,5 @@
   SITE_BRANCH = ""
 [context.deploy-preview.environment]
   SITE_BRANCH = ""
-[context.deploy-deploy.environment]
```
Comment (BotellaA) [inline netlify.toml:5]: Why this change ? It does not work ?

**Review by BotellaA — APPROVED** [review]

File: netlify.toml (hunk 1 above)
Comment (JulienChampagnol) [reply netlify.toml:5]: No, it doesn't work...
